### ISSUE TITLE
Species Inventory/Invasive Assessment

### ISSUE THREAD

**Issue by ElectricGr** [MOJN-Veg/mojn-iu-rpackage#26]

**Comment by ElectricGr** [comment]
Species Inventory

This is a pretty simple survey. It is similar to the previous invasive assessment, except it is now limited to the 50x50m area defined by the transects. This is being combined with an overall species inventory. For native species, we are just recording presence and no other data. For invasive species we are also recording dispersion, Cover, and phenophase. This is the same as we have done previously.

**Comment by ElectricGr** [comment]
Yes, it's in the lookup list excel doc. https://doimspp.sharepoint.com/:x:/r/sites/nps-mojn-data/Shared%20Documents/Projects/2023_IU_Reboot/LookupLists.xlsx?d=w47c57428824346b28a003e0c40f56e11&csf=1&web=1&e=XUyD0u

**Comment by HarpoMark** [comment]
Just published lookups for Dispersion, Invasive Cover, and three soil stability metrics in a desperate attempt to stay ahead of our incredible momentum this week.

**Comment by wright13** [comment]
@ElectricGr Do we need to be able to indicate live vs dead in the inventory?

**Comment by ElectricGr** [comment]
No, we'll only be documenting dead invasives, which is taken care of with the phenophase.

Jeff Galvin
Vegetation Ecologist
Mojave Desert Inventory and Monitoring Network<https://www.nps.gov/im/mojn/index.htm>
National Park Service
601 Nevada Way
Boulder City, NV 89005
Office: 702-293-8667
Cell: 702-286-8646

________________________________
From: Sarah Wright ***@***.***>
Sent: Wednesday, March 29, 2023 3:58 PM
To: MOJN-Veg/mojn-iu-rpackage ***@***.***>
Cc: Galvin, Jeff B ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [MOJN-Veg/mojn-iu-rpackage] Species Inventory/Invasive Assessment (Issue #26)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@ElectricGr<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FElectricGr&data=05%7C01%7Cjeff_galvin%40nps.gov%7Cac39181d3ca74f57cf7608db30a922c5%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C638157275222262918%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3OiyoG28tocrISiO80aVyvixqsWVTYU0ha8duUT7WEw%3D&reserved=0> Do we need to be able to indicate live vs dead in the inventory?

**Comment by ElectricGr** [comment]
@wright13 @HarpoMark 

A few things I noticed. 
-The filter on the species Lu is backwards. Invasives show up when No is selected and native species show up when yes is selected.
-Can you add Unknown plant functionality.

**Comment by wright13** [comment]
Oops, swapped lookups are fixed now. Unknown functionality is there, just needed a tweak to the plants lookup so you should see it now.

**Comment by wright13** [comment]
- [ ] Fix 'other' - don't try to display species

**Comment by ElectricGr** [comment]
@HarpoMark 
Can you add the Dispersion and cover diagrams to the reference on the second page. I put them to NPS-MOJN-Data - Projects\2023_IU_Reboot\Figures
